### PR TITLE
redo undo fixes

### LINKED ISSUE #188: Undo/redo happens letter-by-letter instead of update-by-update
Try changing something with text in it on https://webmaker.mofostaging.net/en-US/goggles. Looks like goggles treats each preview as an undo/redo state.

Would be ideal to just remember states after "update" is clicked.

### PR DESCRIPTION
**Pull request by secretrobotron**
Fixes #188

### CODE REVIEW

**Comment by secretrobotron** [comment]
@Pomax R?

**Comment by secretrobotron** [comment]
1. Removed old (unused) code.
2. Carefully maneuvered around the undo/redo command system

File: webxray/src/mix-master.js [hunk 1]
```diff
@@ -204,7 +219,7 @@
                     // The dialog may have decided to replace all our spaces
                     // with non-breaking ones, so we'll undo that.
                     var html = data.endHTML.replace(/\u00a0/g, " ");
-                    var newContent = self.replaceElement(focusedElement, html);
+                    var newContent = self.replaceElement(originalElement, focusedElement, html, data.finished);
```
Comment (Pomax) [inline webxray/src/mix-master.js]: why do we need to pass in originalElement when it's a direct reference ot focusedElement (rather than a cloned node)?

Comment (secretrobotron) [inline webxray/src/mix-master.js]: On the next line, `focusedElement` changes in this scope every time the `ok` message comes through from the editor dialog. I want to capture the element before the user gets a chance to change it.

Comment (Pomax) [inline webxray/src/mix-master.js]: approved :+1:

**Pull request merged**